README.md

```diff
@@ -70,13 +74,12 @@ yum install rdiff-backup
 ### Debian (From pypi)
 
 ```
-sudo apt update
 sudo apt install python3 python3-setuptools python3-pylibacl python3-pyxattr curl
 curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
I know you didn't change it, but while you're at it, could you install pip3 via apt instead of downloading? I understand, it's packaged.